install/kubernetes/helm/istio/charts/mixer/templates/create-cr-job.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: {{ template "mixer.fullname" . }}-create-crd
+  name: {{ template "mixer.fullname" . }}-create-cr
```

Comment:
why not crd?

Reply:
this create the custom resources not the custom resource definition

Reply:
got it - thanks :)